### DISCUSSION TITLE
[Feature] Support .nomedia / .immichignore patterns

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [immich-app/immich#8578] [Feature Request]
### The feature

It would be amazing to have immich respect .nomedia folders when syncing via the mobile app, or to just have it support glob ignore lists. A nomedia folder is just one which contains a file named .nomedia or a folder whose parents contain such a file. 

![Screenshot_20240406_180206](https://github.com/immich-app/immich/assets/61319150/1a064268-bafb-4279-9107-3dec1687caa3)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by Explosion-Scratch** [comment]
As a side not it would also be great to have sort by name / asset count as well as search in the select albums page.

**Comment by Xingeqwd** [comment]
really, man! 
immich app detected 809 albums on my phone, 
OH, how can i seek the camera, DCIM, screenshot...

thats difficult man, I build the server on WSL with great difficult, I didn’t expect that the client would behave like this....

**Comment by craftyguy** [comment]
I second this. According to the immich android app, there are 1080 albums on my device, and I know for a fact that 99.99% of them are in the `Music` dir (which has a `.nomedia` at the root of it!)... these are `cover.jpg` music album covers that I don't want Immich to deal with *at all*, but the fact that the Immich android app ignores `.nomedia` means that I have to scroll an impossibly long list to try and find the 1 folder I want it to back up.

I have not succeeded in finding this 1 folder yet in the list. 😞

**Comment by Roan-V** [comment]
Same feature request, but a different specific issue here.

The sync local job on my phone is quite slow , and I _suspect_ that this is at least partly caused by the large amount of local albums immich detects: 2884 in fact!

I have quite an extensive music collection that I sync to my phone, it's all nicely structured with artist/album/track.
Most albums have a cover image in their own folder (that alone accounts for 2707 images) each of which immich detects as a local image "album".
My root musiclibrary folder has a .nomedia file, which blocks these images from showing up in most gallery/ file browsers, but not immich sadly.

**Reply by Xingeqwd** [reply]
yes, i think your guess are right...

immich cant support nomedia, that is really a big pity
